flutter/lib/src/sentry_flutter.dart

```diff
@@ -27,13 +26,12 @@ typedef FlutterOptionsConfiguration = FutureOr<void> Function(
 mixin SentryFlutter {
   static const _channel = MethodChannel('sentry_flutter');
 
-  /// Initializes the SDK
   static Future<void> init(
     FlutterOptionsConfiguration optionsConfiguration, {
     AppRunner? appRunner,
-    @internal PackageLoader packageLoader = _loadPackageInfo,
-    @internal MethodChannel channel = _channel,
-    @internal PlatformChecker? platformChecker,
+    PackageLoader packageLoader = _loadPackageInfo,
+    MethodChannel channel = _channel,
+    PlatformChecker? platformChecker,
```

Comment:
What's the reason to remove the method docs and `@internal`?

Reply:
No reason, seems i did not see this change when merging in main.